imports/collections/documentgeneration_private.js

```diff
@@ -0,0 +1,46 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Minutes } from './../minutes';
+import { UserRoles } from './../userroles';
+import { TemplateRenderer } from './../server_side_templates/TemplateRenderer';
+import { DocumentGeneration } from './../documentGeneration';
+
+Meteor.methods({
+    'documentgeneration.createHTML'(minuteID) {
+        //Check DocumentGeneration is enabled and user has rights to continue
+        if (Meteor.settings.public.docGeneration.enabled !== true)
```

Comment:
Please consider putting curly braces around this case (like below in the next `if` clause).